code/game/gamemodes/cult/runes.dm

```diff
@@ -1,1044 +1,944 @@
+		new /mob/living/simple_animal/slaughter/cult(T, pick(NORTH, EAST, SOUTH, WEST))
+		new /mob/living/simple_animal/slaughter/cult(T, pick(NORTHEAST, SOUTHEAST, NORTHWEST, SOUTHWEST))
+		cult_mode.demons_summoned = 1
+		shuttle_master.emergency.request(null, 0.5,null)
+		qdel(src)
 
-	if(istype(src,/obj/effect/rune))
-		usr.say("O bidai nabora se[pick("'","`")]sma!")
-	else
-		usr.whisper("O bidai nabora se[pick("'","`")]sma!")
 
-	if(istype(src,/obj/effect/rune))
-		usr.say("[input]")
+//Rite of Resurrection: Requires two corpses. Revives one and gibs the other.
+/obj/effect/rune/raise_dead
+	cultist_name = "Rite of Resurrection"
+	cultist_desc = "requires two corpses, one on the rune and one adjacent to the rune. The one on the rune is brought to life, the other is turned to ash."
+	invocation = null //Depends on the name of the user - see below
+	icon_state = "1"
+
+/obj/effect/rune/raise_dead/invoke(var/list/invokers)
+	var/turf/T = get_turf(src)
+	var/mob/living/mob_to_sacrifice
+	var/mob/living/mob_to_revive
+	var/list/potential_sacrifice_mobs = list()
+	var/list/potential_revive_mobs = list()
+	var/mob/living/user = invokers[1]
+	if(rune_in_use)
+		return
+	for(var/mob/living/M in orange(1,src))
+		if(M.stat == DEAD  && !iscultist(M))
+			potential_sacrifice_mobs.Add(M)
+	if(!potential_sacrifice_mobs.len)
+		to_chat(user, "<span class='cultitalic'>There are no eligible sacrifices nearby!</span>")
+		log_game("Raise Dead rune failed - no catalyst corpse")
+		return
+	mob_to_sacrifice = input(user, "Choose a corpse to sacrifice.", "Corpse to Sacrifice") as null|anything in potential_sacrifice_mobs
+	if(!Adjacent(user) || !src || qdeleted(src) || user.incapacitated() || !mob_to_revive || !mob_to_sacrifice || rune_in_use)
+		return
+	for(var/mob/living/M in T.contents)
+		if(M.stat == DEAD)
+			potential_revive_mobs.Add(M)
+	if(!potential_revive_mobs.len)
+		to_chat(user, "<span class='cultitalic'>There is no eligible revival target on the rune!</span>")
+		log_game("Raise Dead rune failed - no corpse to revived")
+		return
+	mob_to_revive = input(user, "Choose a corpse to revive.", "Corpse to Revive") as null|anything in potential_revive_mobs
+	if(!Adjacent(user) || !src || qdeleted(src) || user.incapacitated() || rune_in_use)
+		return
+	if(!in_range(mob_to_sacrifice,src))
+		to_chat(user, "<span class='cultitalic'>The sacrificial target has been moved!</span>")
+		fail_invoke()
+		log_game("Raise Dead rune failed - catalyst corpse moved")
+		return
+	if(!(mob_to_revive in T.contents))
+		to_chat(user, "<span class='cultitalic'>The corpse to revive has been moved!</span>")
+		fail_invoke()
+		log_game("Raise Dead rune failed - revival target moved")
+		return
+	if(mob_to_sacrifice.stat != DEAD)
+		to_chat(user, "<span class='cultitalic'>The sacrificial target must be dead!</span>")
+		fail_invoke()
+		log_game("Raise Dead rune failed - catalyst corpse is not dead")
+		return
+	rune_in_use = 1
+	if(user.name == "Herbert West")
```

Comment:
i can remove it but its a kinda funny reference esicpally if you know the musical https://www.youtube.com/watch?v=OJAoaCHdTJY